libs/utils/energy_model.py

```diff
@@ -830,7 +830,18 @@ def from_simplifiedEM_target(cls, target,
             sysfs_em[fd]['cpus'] = cpus
             sysfs_em[fd]['frequency'] = map(int, sysfs_em[fd]['frequency'].split(' '))
             sysfs_em[fd]['power'] =  map(int, sysfs_em[fd]['power'].split(' '))
-            sysfs_em[fd]['capacity'] = map(int, sysfs_em[fd]['capacity'].split(' '))
+
+            if 'capactity' in sysfs_em[fd]:
```

Comment:
Hmm thinking about it, do we really want to support <v4 bindings? Or do you want to keep it as it to merge it now, and eventually remove the v4 bits?

Reply:
Actually I can just remove that and just recompute the 'capacity' no matter what the EM version is. That'll work just fine for both versions